src/components/Layout/Header/NavBar/Native/BackupPassphraseModal/BackupPassphrasePassword.tsx

```diff
@@ -59,6 +58,7 @@ export const BackupPassphrasePassword: React.FC<LocationState> = props => {
   const handleShowClick = useCallback(() => setShowPw(!showPw), [showPw])
   const onSubmit = useCallback(
     async (values: FieldValues) => {
+      const Vault = await import('@shapeshiftoss/hdwallet-native-vault').then(m => m.Vault)
```

Comment:
🧠  - we probably want to `React.lazy` the route components as well in `<BackupPassphraseRouter />` to ensure the whole tree is lazy-loaded, though this can be done in a follow-up!